### PR TITLE
libcflib

### PR DESCRIPTION
**Pull request by scopatz**
new noarch package

### CODE REVIEW

File: recipes/libcflib/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE.txt
+  summary: 'Library Conda Forge Library '
+
+extra:
+  recipe-maintainers:
+    - scopatz
+    - CJ-Wright
+    - mariusvniekerk
```
Comment (scopatz) [inline recipes/libcflib/meta.yaml]: @CJ-Wright @mariusvniekerk - are you OK with being maintainers?

Comment (CJ-Wright) [reply recipes/libcflib/meta.yaml]: Yes of course! :smile:

Comment (mariusvniekerk) [reply recipes/libcflib/meta.yaml]: Sure

**Comment by mariusvniekerk** [comment]
Looks like we are missing some things in the manifest.in

File: recipes/libcflib/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
+{% set name = "libcflib" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```
Comment (CJ-Wright) [inline recipes/libcflib/meta.yaml]: have we not released a rever tarball?

File: recipes/libcflib/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  imports:
+    - libcflib
+    - libcflib.environ
+
+about:
+  home: http://github.com/regro/libcflib
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE.txt
+  summary: 'Library Conda Forge Library '
```
Comment (CJ-Wright) [inline recipes/libcflib/meta.yaml]: extra space after library

**Comment by scopatz** [comment]
This is probably good to go now!

**Pull request merged**